api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -79,6 +79,79 @@ export function toSARoleAssignmentBody(currentUserId: string, allocateRoleData:
   };
 }
 
+export function toDenySARoleAssignmentBody(currentUserId: string, allocateRoleData: AllocateRoleData): any {
+  return {
+    roleRequest: {
+      assignerId: currentUserId,
+      replaceExisting: true,
+      process: 'specific-access',
+      reference: `${allocateRoleData.caseId}/${allocateRoleData.requestedRole}/${allocateRoleData.person.id}`,
+    },
+    requestedRoles: [{
+      roleType: 'CASE',
+      readOnly: true,
+      grantType: 'BASIC',
+      classification: 'PRIVATE',
+      attributes: {
+        caseId: allocateRoleData.caseId,
+        requestedRole: allocateRoleData.requestedRole,
+      },
+      roleName: allocateRoleData.typeOfRole.id,
+      roleCategory: allocateRoleData.roleCategory,
+      actorIdType: 'IDAM',
+      actorId: allocateRoleData.person.id,
+      endTime: new Date(new Date().setDate(new Date().getDate() + 14)),
+      beginTime: new Date(),
+      notes: [{comment: allocateRoleData.comment,
+      time: "2022-05-10T16:34:18.763Z",
+      userId: allocateRoleData.person.id}],
+
+      // NOTE: There is not information getting previously request role at the moment on LLD documantation,
+      // and it s not included in steps on requirement
+      // when it's ready originalRequestJustification and originalRequestDate will be replaced with it
+      originalRequestDate: "2022-05-22T16:34:18.763Z",
+      originalRequestJustification: {
+        roleRequest: {
+          assignerId: 'db17f6f7-1abf-4223-8b5e-1eece04ee5d8',
+          process: 'specific-access',
+          reference: '1613568559071553/specific-access-legal-operations/db17f6f7-1abf-4223-8b5e-1eece04ee5d8',
+          replaceExisting: true,
+        },
+        requestedRoles: [
+          {
+            actorIdType: 'IDAM',
+            actorId: 'db17f6f7-1abf-4223-8b5e-1eece04ee5d8',
+            roleType: 'CASE',
+            roleName: 'specific-access-requested',
+            classification: 'PRIVATE',
+            roleCategory: 'LEGAL_OPERATIONS',
+            grantType: 'BASIC',
+            beginTime: null,
+            endTime: '2022-06-21T12:34:09.101Z',
+            attributes: [Object],
+            notes: [Array],
+            readOnly: true,
+          },
+        ],
+      },
+    },
+  ],
+  };
+}
+export function toDenySADletionRequestedRoleBody(requestId: string): any {
+  return {
+      pathVariables: {
+        process : 'staff-organisational-role-mapping',
+        reference : requestId,
+      },
+      queryParams : null,
+      body: {
+        userIds : [ requestId ],
```

Comment:
Minor comment - can we remove the inconsistent spacing between the fields and the ':' please?